wordle-web/components/UserInput.vue

```diff
@@ -0,0 +1,35 @@
+<template>
```

Comment:
great implementation for getting the username.  May want add a button or click-able box to show the current username and allow for it to be changed.

Reply:
Thank you, it is done now.